pkg/sdk/symbols/extract/async.go

```diff
@@ -35,8 +35,8 @@ const (
 )
 
 const (
-	starvationThresholdNs = 1e6
-	sleepTimeNs           = 1e4 * time.Nanosecond
+	starvationThresholdNs = int64(1e6)
+	sleepTimeNs           = 1e7 * time.Nanosecond
```

Comment:
Sleeping a little bit more brings down CPU usage from ~15% to ~3%. Since extraction-intensive use cases will not trigger the starvation, increasing the sleeping time ensure a lower CPU usage when the worker is idle.